docs/queries.rst

```diff
@@ -0,0 +1,232 @@
+Queries & ObjectTypes
+=====================
+
+Introduction
+------------
+
+The core functionality of GraphQL is to make queries in order to get data.
```

Comment:
I don't think this sentence is adding much. Maybe "Graphene offers a host of features when performing GraphQL queries."